server.js

```diff
@@ -28,4 +28,27 @@ app.post('/upload', (req, res) => {
     });
 });
 
+app.get('/use/:file', (req, res) => {
+    if (req.params.file === null) {
+        return res.status(400).json({ msg: 'No file requested' });
+    }
+
+    var filePath = `${__dirname}/uploads/${req.params.file}`;
```

Comment:
This is a really minor thing, but you should use `let` or `const` instead of `var`